app/server/src/main/scala/org/bitcoins/server/WalletRoutes.scala

```diff
@@ -54,6 +54,20 @@ case class WalletRoutes(wallet: AnyHDWalletApi)(implicit
         }
       }
 
+    case ServerCommand("walletinfo", _) =>
+      complete {
+        wallet.getDefaultAccount().map { accountDb =>
+          val json = Obj(
+            "keyManager" -> Obj(
```

Comment:
so here is my commit that you can apply to your branch to do some things I would like that help my OCD. If you look carefully here we are using a different configuration key `keyManager` rather than `keymanager` here. 

We should really try to use the same keys for everything other wise it will drive us slowly insane 

https://github.com/Christewart/bitcoin-s-core/commit/7674a3c44ac3958ac4b52f6d574589ab02ca5ffe

Eventually it would be nice to have `getInfo` for every project and then we can nicely compose them for a `getinfo` in the `appServer` project that has information about all of our sub modules.